yaksh/static/yaksh/css/exam.css

```diff
@@ -1,7 +1,13 @@
-table td, table th { border: #ff8295 solid 1px !important;
+#assertion td, #assertion th, #stdio td, #stdio th { border: #ff8295 solid 1px !important;
 			word-wrap: break-word !important;
 			white-space: pre-wrap !important;
 			}
 #stdio, #assertion {
 	table-layout: fixed
 }
+.table{
```

Comment:
Give a different name if custom CSS is needed!
Because table class provided by bootstrap will overwritten if used by any one else in the same file in future.